samples/samples-csharp/Database/Tables/ProductsWithReservedPrimaryKeyColumnNames.sql

```diff
@@ -0,0 +1,14 @@
+﻿CREATE TABLE [ProductsWithReservedPrimaryKeyColumnNames] (
```

Comment:
These should go in https://github.com/Azure/azure-functions-sql-extension/tree/main/test/Integration/test-csharp, the samples folder should be only for valid samples that we want customers to be looking at for their own purposes. Objects used for negative/failure tests should go in the test folder. 

(you'll probably have to update the logic to look for the SQL files in both places though, I don't think we do that currently since all our current tests use the sample tables)